### ISSUE TITLE
Series.view without specified dtype

### ISSUE THREAD

**Issue by jorisvandenbossche** [pandas-dev/pandas#20281]
xref https://github.com/pandas-dev/pandas/pull/20220/files#r173655704

Currently, `Series.view()` is actually equivalent to `Series.view('float64')` (because that is the default dtype in the Series constructor) which often does not make sense. Not too important (as there are not many reasons to use this function in pandas), but I think we should fix to either a) disallow `dtype=None` or either b) if dtype is None to set `dtype = self.dtype` (behaviour of numpy if no dtype is specified)

**Comment by jreback** [comment]
ok, this is fine, though we should simply deprecate this.

**Comment by villasv** [comment]
I don't think this function is going to be used a lot, but I wouldn't mind leaving it there instead of deprecating it. I'm specially in favor of the second solutution (if dtype is None to set dtype = self.dtype)

**Comment by jreback** [comment]
@villasv see my comment son the other issue
this is pretty useful in the current memory model, causes bugs, is not tested and you can simply do this with .astype or use a numpy view directly if you want

**Comment by villasv** [comment]
Sound arguments, I'm in favor of keeping it but not specially attached to it. I won't miss it if it's gone. I'll try to keep discussions in the other issue only.

**Comment by mroeschke** [comment]
Looks like this respects the original dtype now, so maybe just need to update the docs

**Comment by jorisvandenbossche** [comment]
We can indeed certainly fix the docs, but IMO we should just deprecate it as well -> https://github.com/pandas-dev/pandas/issues/20251

**Comment by jorisvandenbossche** [comment]
The `view()` method has been deprecated now, so going to close this issue.
